src/js/components/Drop/DropContainer.js

```diff
@@ -225,6 +233,7 @@ const DropContainer = forwardRef(
             container.style.maxHeight = `${maxHeight}px`;
           }
         }
+        updateAlignControlMirror();
```

Comment:
Can we move the `if (onAlign)` check here, since it won't do anything unless the caller passes it in?

Reply:
Agreed! Addressed!